### PR TITLE
docs: add i18n and trailing slash description file

### PR DESCRIPTION
**Pull request by DammneD**

### CODE REVIEW

File: docs/routing/localisation_and_trailing_slash.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+## Trailing slash
+
+A trailing slash is a forward slash (“/”) placed at the end of a URL.
+The decision to use a trailing slash or not is more a personal preference than actual necessity, except for some special cases. In any case we provide the ability to control this.
+
+![Trailing slash](../assets/routes/trailing-slash.png)
+
+This menu has 3 options:
+
+ ![Trailing slash settings](../assets/routes/trailing-slash-settings.png)
```
Comment (StyleT) [inline docs/routing/localisation_and_trailing_slash.md:33]: Pls rm, it's excessive here

File: docs/routing/localisation_and_trailing_slash.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## i18n
+
+ILC comes with built in internationalization support.
+
+![i18n screenshot](../assets/routes/i18n.png)
+
+While having i18n feature enabled - ILC allows you to handle localized URLs at global level. Currently, we support two routing schemes which can be switched in the `i18n.routingStrategy` menu:
+
+![i18n routing strategy](../assets/routes/i18n-settings.png)
```
Comment (StyleT) [inline docs/routing/localisation_and_trailing_slash.md:11]: Pls rm, it's excessive here

**Review by StyleT — CHANGES_REQUESTED** [review]

**Review by StyleT — APPROVED** [review]

File: docs/routing/localisation_and_trailing_slash.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Here's what it looks like in practice:
+
+![i18n language prefix example](../assets/routes/language-prefix.png)
+
+More details about i18n you can read [here](../i18n.md)
+
+## Trailing slash
+
+A trailing slash is a forward slash (“/”) placed at the end of a URL.
+The decision to use a trailing slash or not is more a personal preference than actual necessity, except for some special cases. In any case we provide the ability to control this.
```
Comment (bandantonio) [inline docs/routing/localisation_and_trailing_slash.md:25]:
```suggestion
ILC allows you to manage cases with a trailing slash, but the decision to use it is at your discretion, considering personal preference and special cases you may have.
```

File: docs/routing/localisation_and_trailing_slash.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+![Trailing slash](../assets/routes/trailing-slash.png)
+
+This menu has 3 options:
```
Comment (bandantonio) [inline docs/routing/localisation_and_trailing_slash.md:29]:
```suggestion
To manage trailing slash behavior, there are the following options:
```

File: docs/routing/localisation_and_trailing_slash.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+ 1. **Do nothing** - Doesn't affect the URL in any way.
+ 2. **Redirect to non trailing slash** - Removes slash at the end of the URL if it was there.
+ 3. **Redirect to trailing slash** - Adds a slash to the end of the URL if it wasn't there.
```
Comment (bandantonio) [inline docs/routing/localisation_and_trailing_slash.md:33]:
```suggestion
1. **Do nothing** - No effect at URLs.
1. **Redirect to non-trailing slash** - Remove trailing slash at the end of URLs, if present.
1. **Redirect to trailing slash** - Add trailing slash to the end of URLs, if not present.
```

File: mkdocs.yml [hunk 6]
```diff
@@ -40,6 +40,7 @@ nav:
     - Routing:
       - Route configuration: docs/routing/route_configuration_options.md
       - Match & cascade merge: docs/routing/route_match_patterns.md
+      - i18n & trailing slash: docs/routing/localisation_and_trailing_slash.md
```
Comment (bandantonio) [inline mkdocs.yml:43]:
```suggestion
      - i18n & trailing slash: docs/routing/localization_and_trailing_slash.md
```

Please rename the markdown file accordingly.

**Review by bandantonio — CHANGES_REQUESTED** [review]

**Review by bandantonio — APPROVED** [review]

**Review by nightnei — APPROVED** [review]

**Pull request merged**